java/ql/src/experimental/Security/CWE/CWE-036/OpenStream.ql

```diff
@@ -0,0 +1,56 @@
+/**
+ * @name Calling openStream on URLs created from remote source can lead to file disclosure
```

Comment:
Generally we try to use a shorter `@name` but this would be perfect for a description.

Perhaps: 
```suggestion
 * @name openStream called on URLs created from remote source
```

Reply:
Hi @felicitymay, thanks for the feedback!